defaults/main.yml

```diff
@@ -5,3 +5,6 @@
 base_debian_mirror: http://deb.debian.org/debian
 # Debian backports are disabled by default
 base_enable_backports: false
+# The "ntp" package used to be our default ntp client / daemon.
+# Set the ntp package to be installed (e.g. chrony or systemd-timesyncd)
+base_ntp_client: "ntp"
```

Comment:
The `client` within `base_ntp_client` feels wrong to me, because it's really about the actual *daemon* that should be present, nor? Maybe just name it `base_ntp_package` or so?